src/index.njk

```diff
@@ -13,79 +13,88 @@ masthead: true
+        <p class="govuk-body govuk-!-margin-bottom-0">
+          <a href="/components/" class="govuk-link govuk-!-font-weight-bold">Browse our components</a>
+        </p>
       </div>
       <div class="govuk-grid-column-one-third-from-desktop govuk-!-margin-bottom-8">
         <h2 class="govuk-heading-l">Patterns</h2>
         <p class="govuk-body">Help users complete common tasks like entering names and addresses, filling in forms and creating accounts.</p>
-        <p class="govuk-body govuk-!-margin-bottom-0"><a href="/patterns/" class="govuk-link govuk-!-font-weight-bold">Browse our patterns</a></p>
+        <p class="govuk-body govuk-!-margin-bottom-0">
+          <a href="/patterns/" class="govuk-link govuk-!-font-weight-bold">Browse our patterns</a>
+        </p>
       </div>
+    </div>
 
-      <div class="govuk-grid-column-full">
-        <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl govuk-!-margin-top-0">
-      </div>
+    <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl govuk-!-margin-top-0">
 
-      <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds-from-desktop">
         <h2 class="govuk-heading-l">Principles we follow</h2>
         <p class="govuk-body">
-The GOV.UK Design System helps teams that work on government services follow the <a class="govuk-link" href="https://www.gov.uk/guidance/government-design-principles">Government Design Principles</a> and <a class="govuk-link" href="https://www.gov.uk/service-manual">GOV.UK Service Manual</a>. This website also follows the <a class="govuk-link" href="https://www.gov.uk/guidance/style-guide">GOV.UK style guide</a> and <a class="govuk-link" href="https://www.gov.uk/guidance/content-design">GOV.UK content design guidance</a>.
+          The GOV.UK Design System helps teams that work on government services follow the
+          <a class="govuk-link" href="https://www.gov.uk/guidance/government-design-principles">Government Design Principles</a> and
+          <a class="govuk-link" href="https://www.gov.uk/service-manual">GOV.UK Service Manual</a>. This website also follows the
+          <a class="govuk-link" href="https://www.gov.uk/guidance/style-guide">GOV.UK style guide</a> and
+          <a class="govuk-link" href="https://www.gov.uk/guidance/content-design">GOV.UK content design guidance</a>.
         </p>
 
-      <p class="govuk-body">
-<a class="govuk-link" href="/accessibility/accessibility-strategy/">Our accessibility strategy</a> outlines our principles and work to improve accessibility within the GOV.UK Design System.</p>
+        <p class="govuk-body">
+          <a class="govuk-link" href="/accessibility/accessibility-strategy/">Our accessibility strategy</a> outlines our principles and work to improve accessibility within the GOV.UK Design System.
+        </p>
       </div>
+    </div>
```

Comment:
Do we lint this page? Was it passing the lint by fudging this closing tag somehow?

Reply:
Ah, I see now, I got caught in a  landslide of `<div>`s